src/reanimated2/animation/spring.ts

```diff
@@ -112,6 +136,7 @@ export function withSpring(
       animation: SpringAnimation
     ) {
       return (
+        previousAnimation?.startTimestamp &&
```

Comment:
I ask just in case 😅 Do you know that, when `startTimestamp` will be equal `0` then `previousAnimation?.startTimestamp = false` ?

Reply:
Yes I do, we have already identical assertion in `timing.ts`